Docs/developer_guide/python_faq.md

```diff
@@ -296,6 +296,16 @@ In most places, unix-type separators can be used instead of backslash. This is c
 
 See more information in Python documentation: https://docs.python.org/3/tutorial/introduction.html?#strings
 
+## How to modify a Python scripted module
+
+If `Developer mode` is enabled in application settings then `Reload and Test` section is displayed at the top of user interface of Python scripted modules. This section contains buttons for convenient editing of the module source code (`.py` file) and user interface (`.ui` file). Clicking the `Edit` button opens the module source code in the program associated with `.py` files, as defined in operating system settings. This behavior can be overridden by specifying a text editor (such as VS Code, Sublime Text, ...) in application settings: `Editor for .py files` in `Python` section.
```

Comment:
```suggestion
If the `Developer mode` is enabled in the application settings then a `Reload and Test` section is displayed at the top of the user interface of Python scripted modules. This section contains buttons for convenient editing of the module source code (`.py` file) and user interface (`.ui` file). By default, clicking on the `Edit` button opens the module source code in the program associated with `.py` files, as defined in the operating system settings. This behavior can be overridden by specifying a text editor (such as VS Code, Sublime Text, ...) in the application settings: `Editor for .py files` in the `Python` section.
```

Reply:
Thanks for the suggestions @jcfr. An your explain why "the" is needed in "If the `Developer mode` is enabled"?

Reply:
It initially felt more natural ... and your question prompted me to further look into it. See https://learnenglish.britishcouncil.org/grammar/english-grammar-reference/definite-article

Looking at this page `Developer Mode` is use without `the`.
See https://developer.apple.com/documentation/xcode/enabling-developer-mode-on-a-device

Looking at the current Slice code base, there is one occurrence of `when the [developer mode]...` and all the other ones are `developer mode` all lower cased (unless at the start of a sentence)

Most of the occurrences at `developer mode` with a few `Developer mode`.

For sake of consistency with how we define module panel[^1], I think we should use `Developer Mode`

[^1]: https://www.slicer.org/wiki/Documentation/Nightly/Developers/Style_Guide/UI#Text